src/test/resources/feature/BDDscenarioForWishList.feature

```diff
@@ -0,0 +1,46 @@
+@WishList
```

Comment:
Remove this tag, you must have as few tags as possible. Do this for the rest of tags. Ideal will be if you will have hust one runner which will run teste by your name tag.

Reply:
done